src/app/chat/views/message_format.nim

```diff
@@ -15,30 +15,44 @@ proc mention(self: ChatMessageList, pubKey: string): string =
 
 # See render-inline in status-react/src/status_im/ui/screens/chat/message/message.cljs
 proc renderInline(self: ChatMessageList, elem: TextItem): string =
-  let value = escape_html(elem.literal.strip)
+  let value = escape_html(elem.literal)
   case elem.textType:
   of "": result = value
-  of "code": result = fmt(" <code>{value}</code> ")
-  of "emph": result = fmt(" <em>{value}</em> ")
-  of "strong": result = fmt(" <strong>{value}</strong> ")
-  of "link": result = fmt(" {elem.destination} ")
-  of "mention": result = fmt(" <a href=\"//{value}\" class=\"mention\">{self.mention(value)}</a> ")
-  of "status-tag": result = fmt(" <a href=\"#{value}\" class=\"status-tag\">#{value}</a> ")
+  of "code": result = fmt("<code>{value}</code>")
+  of "emph": result = fmt("<em>{value}</em>")
+  of "strong": result = fmt("<strong>{value}</strong>")
+  of "link": result = fmt("{elem.destination}")
+  of "mention": result = fmt("<a href=\"//{value}\" class=\"mention\">{self.mention(value)}</a>")
+  of "status-tag": result = fmt("<a href=\"#{value}\" class=\"status-tag\">#{value}</a>")
+  of "del": result = fmt("<del>{value}</del>")
 
 # See render-block in status-react/src/status_im/ui/screens/chat/message/message.cljs
 proc renderBlock(self: ChatMessageList, message: Message): string =
   for pMsg in message.parsedText:
     case pMsg.textType:
       of "paragraph": 
         result = result & "<p>"
-        if message.isCurrentUser:
-          result = result & "&nbsp;"
         for children in pMsg.children:
           result = result & self.renderInline(children)
         result = result & "</p>"
       of "blockquote":
-        result = result & pMsg.literal.strip.split("\n").mapIt("<span>▍ " & escape_html(it) & "</span>").join("<br />")
+        var
+          blockquote = escape_html(pMsg.literal)
+          lines = blockquote.split("\10")
```

Comment:
I'm wondering if this would create problems. For example, if you reply to someone and in your reply you put `2 < 3`
as a mathematical solution or whatever, then it would split there I think?
Maybe we should split on `\n\10`?
(this is all assuming `>` is `\10` haha. I have no idea

Reply:
`\10` is a line break that is entered with `Shift + Enter`, which is the only way to enter a line break while typing without sending the message (on macos at least). It's possible this is different on other OS's and needs to be taken in to account.

Reply:
Might need to see if this is different with other OS's... maybe you could take this branch for a spin and type the following in to the chat:
```
> this is a quote
```
Then press `Shift+Enter` to add a new line. It should prepend the next line with `> `. Then add `this is the second line`:
```
> this is a quote
> this is the second line
```
Then press `Shift+Enter` twice, it should then clear the blockquote, like this:
```
> this is a quote
> this is the second line

|        <== cursor should be here
```
Then add `this is a reply`:
```
> this is a quote
> this is the second line

this is a reply
```
Then press `Enter` to send the message.

It should appear like this:
![Imgur](https://imgur.com/eCQWTWu.png)

Reply:
I updated this to a regex match of `\n|\r` which seems to be what we're doing in other places, to hopefully cover line returns on all platforms. Still would be good if you could try this out.

Reply:
Update: @jrainville tested this and it works on Windoze. Yay!